12_fizzbuzz.rb

```diff
@@ -26,4 +26,19 @@
 # ...etc
 #
 
-# Your code here  
+def fizzbuzz
+  range = (1..100).to_a
+  range.each do |x|
```

Comment:
Awesome solution.

You could also ditch the array and:
`(1..100).times do |x|`